addons/website/views/snippets/s_table_of_content.xml

```diff
@@ -10,7 +10,7 @@
                 </div>
                 <div class="col-lg-9 s_table_of_content_main oe_structure oe_empty" data-name="Content">
                     <section class="pb16">
-                        <h1 data-anchor="true">Intuitive system</h1>
+                        <h2 class="h1-fs" data-anchor="true">Intuitive system</h2>
```

Comment:
See here you use `h1-fs` and not `h1`... :grimacing: We need a choice of what to use in snippets.

Reply:
Note that this could be judged weird too: the UI display h2 but nothing explain why changing h2 in the theme tab does not change the size of this :grimacing:

Reply:
It's true but I think it's OK. It was already the case before and AFAIK nobody ever complains about that. The only solution to fix that weird behavior would be to have only text style and don't propose custom font sizes which sound less good.

But that make me think that maybe, if we have `<h1 class="display-3-fs">TEXT</h1>` and the user clicks on the default font size, we could just removes the `display-3-fs` class instead of forcing the base font size. I don't know what the user expect in this case... :thinking:

Reply:
> it was already the case before

Not sure what you mean, there was nothing before: no use of those classes in snippets and no possibility to change the font-size in the right panel. So there was no weird behavior. Less possibility but nothing weird.

But it's ok, we can live with either h1 of h1-fs in this case I recon. One is weird for font family and color, the other is weird for font-size... we can still think about it post-merge.

In fact I would rather let is be a standard h2, no class in this specific case.

> the user clicks on the default font size, we could just removes the display-3-fs class instead of forcing the base font size

"Default" should be renamed to "Normal", just like it is in the text style dropdown. It's next to "16" which means, "normal default text size". If you want the "default for an h1", just click on the header 1 font **style**.